.github/workflows/link-check-all.yml

```diff
@@ -5,4 +5,4 @@ on:
     - cron: '0 0 * * *'
 jobs:
   run:
-    uses: iterative/link-check/.github/workflows/link-check-all.yml@v0.13.0
+    uses: iterative/link-check/.github/workflows/link-check-all.yml@v0.13.1
```

Comment:
can we just do `v0` rather than `v0.13.1` to make it auto-update? Or is there already some easy process to bump versions?